src/binaries/table-meta/main.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 Datafuse Labs.
```

Comment:
Better to name `src/binaries/tools/table_meta_inspector.rs`, then we can add others tool to here in future.

Reply:
get it!